### DISCUSSION TITLE
More flexibility to edit the design of checkout components

### DISCUSSION THREAD

**Discussion by juneyuanjunliu** [woocommerce/woocommerce#53875] [Checkout Flow]
Currently just being able to add custom fields doesn't do much work to increase my store's conversions. I need better flexibility in adding blocks like timer, icons, and being able to change the design of the components in Checkout Block.

As well as the ability to change the final order confirmation page layout in a simple Kadence block editor interface.

**Comment by ralucaStan** [comment]
Hi @juneyuanjunliu,
we'd like to lear more about the customizations you want to bring. What are the features you have in mind when talking about adding adding blocks like timer & icons? 

Could you provide more details about the order confirmation change?
